docs/content/tutorial/index.md

```diff
@@ -0,0 +1,34 @@
+# Apache ZooKeeper charmed operator tutorial
+
+The Apache ZooKeeper charmed operator simplifies the deployment, management, and scaling of Apache ZooKeeper clusters on your Juju environment. 
+
+In this tutorial, we will:
+
+* Set up your environment using LXD and Juju.
+* Deploy Apache ZooKeeper using charmed operator.
+* Integrate Apache ZooKeeper and Apache Kafka charms.
+
+While this tutorial intends to teach you to use Apache ZooKeeper charmed operator, it will be most beneficial if you already have a familiarity with:
+
+* Basic terminal commands
+* [Apache Zookeeper](https://zookeeper.apache.org/) concepts
+
+## Minimum system requirements
+
+Before we start, make sure your machine meets the following requirements:
+
+* Ubuntu 22.04 (Focal) or later.
+* 8 GB of RAM.
```

Comment:
I believe we can reduce this to even 2GB for ZooKeeper. Taking it from the ZK docs:

> ZooKeeper is usually deployed on dedicated RHEL boxes, with dual-core processors, 2GB of RAM, and 80GB IDE hard drives.

So if we use 3 ZK, we should use 6GB. If we have 5, we should have 10GB